functests/ptp/ptp.go

```diff
@@ -108,4 +96,437 @@ var _ = Describe("ptp", func() {
 			}
 		})
 	})
+	var _ = Describe("PTP e2e tests", func() {
+		var runningPod *v1core.PodList
```

Comment:
Can this be runningPods? Also, can this be local so you can have only ptpRunningPods as global and use it in line 176, as you are applying the same filters